### ISSUE TITLE
Send cron-based notifications

### ISSUE THREAD

**Issue by lindseydiloreto** [doublesecretagency/craft-notifier#10]
Paraphrased from Discord:

> _"... check entries' expiration date, and if it's going to expire within X days, send a notification"_

It would be great if notifications could be triggered via a normal cron job.

The user could set a simple cron to point to a Controller endpoint.

**Comment by lindseydiloreto** [comment]
This would also come in handy for monitoring an RSS feed, such as this feed of Craft Critical Updates:

- https://feeds.craftcms.com/critical.atom

**Comment by brianrivet-tilt** [comment]
I don't know if this falls under this as well, but it would be good if you want to do notifications that are tied to an entry posting, (vs. it being saved) it would be good to have the notification go out when the entry posts (for instance if you set a post date in the future) rather than when it is saved (or have the option for either)

**Comment by lindseydiloreto** [comment]
Good call. I'm not sure yet whether or not they're related, but thanks for making a note about it!

**Comment by lindseydiloreto** [comment]
Great news, all of these new event triggers have been officially released in [v3.0.0](https://github.com/doublesecretagency/craft-notifier/releases/tag/3.0.0). 🎉 

Per the original feature request, Notifier can now send a message [when a scheduled date is reached](https://plugins.doublesecretagency.com/notifier/events/types/entries/date-reached).

And per @brianrivet-tilt's request, it can also send a message [when an entry changes from Pending to Live](https://plugins.doublesecretagency.com/notifier/events/types/entries/pending-to-live).

Lastly, it can also send a message by [watching an RSS/JSON feed](https://plugins.doublesecretagency.com/notifier/events/types/feed/).

Hope these new event types are helpful! 👏
